popup.js

```diff
@@ -0,0 +1,251 @@
+var axios = require("axios");
```

Comment:
this file is too large .. that's ok since this is your first project,, but please read up on **separation of concerns (SoC)** here are a few resources:
http://deviq.com/separation-of-concerns/
https://stackoverflow.com/questions/98734/what-is-separation-of-concerns
http://www.codeconquest.com/how-the-separation-of-concerns-makes-programming-magic/